templates/admin/components/_navbar.html.twig

```diff
@@ -11,6 +11,9 @@
             <li class="nav-item">
                 <a class="nav-link" href="{{ path('admin_user_index') }}">Utilisateurs</a>
             </li>
+            <li class="nav-item">
+                <a class="nav-link" href="{{ path('admin_user_unactivate') }}">Utilisateurs non vérifié</a>
```

Comment:
"vérifiés"

It would be nice to have a button that delete all the users at the same time and/or to redirect the admin to the same list of unverified users.